packages/esm-patient-test-results-app/src/hiv/concepts.ts

```diff
@@ -0,0 +1,58 @@
+const concepts = {
+  display: 'Bloodwork',
+  concept: 'ConvSet', // convienence set
```

Comment:
Since you already have the `display`, are you using `concept: 'ConvSet'` for anything?

Reply:
No, it doesn't look like `concept` is currently being used. I will delete it.

Reply:
Although, It might help later if we want to group findings from a convenience set together on the timeline. Is this difficult to provide also? I think this is cheap and might provide some good benefits for FE if it is existing.

Reply:
You got it right that it is cheap to add. But given the fact that we are developing for resource constrained environments, where the more data we return to the client, the more client side memory and bandwidth we consume, i would like to return only the bare minimum that you need for now. We can always easily add new fields as we need them. On the backend, am implementing this in the REST webservices module which has more flexible and rapid release cycles than the core platform.

Reply:
Ok, let's drop it for now and think about adding later